src/Raven.Client/ServerWide/DatabaseRecord.cs

```diff
@@ -70,6 +71,12 @@ public class ShardRangeAssignment
             public int Shard;
         }
 
+        public Dictionary<int, BucketMigration> BucketMigrations;
+
+        public long MigrationCutOffIndex;
+
+        public string ShardedDatabaseId;
```

Comment:
What is that for?

Reply:
this is for the "dbid" part of the `MOVE` tag in the migrated change vector.